packages/main/src/AINotice.hbs

```diff
@@ -0,0 +1,37 @@
+<div class="ainotice">
```

Comment:
all class names should be changed - see some existing components for the conventions